README.md

```diff
@@ -73,12 +73,16 @@ Config has a following format:
 
 ## Releasing new versions
 
-In order to create a release, manully update the version in `package.json` of your feature branch.
-You can also use [`npm version`](https://docs.npmjs.com/cli/version) to do so.
-
-Add details about changes in `Changelog.md`.
-
-New versions are automatically published to Gemfury on every merge to `master`.
+In order to create a release:
+
+1. Update the version in `package.json` of your feature branch
+    1. You can also use [`npm version`](https://docs.npmjs.com/cli/version) to do so
```

Comment:
npm version will add a commit and tag with the right version so step 3(?) wouldn't be required if you do this
Also all steps are 1 😄

Reply:
everything is important :P

Reply:
👍  I've updated it to be more obvious

Reply:
Okay looks good but they are all still step 1 😅

Reply:
@unsign3d @mattp91 Using all 1s is a feature of Markdown. It resolved to be correctly counted - https://github.com/babbel/miza-kinesis/blob/release-1.0.9-again/README.md#releasing-new-versions, which is useful because you don't need to update every number when you add a new step.

Reply:
Interesting, I didn't know that 😅  still looks odd when reading the markdown but okay :)